samples/HCL/example.tf

```diff
@@ -1,13 +1,145 @@
-resource "aws_instance" "web" {
```

Comment:
No worries, I've put reverted the change to `example.tf`. I've also added `main.tf`, which is a configuration used to deploy [Dokku](https://github.com/dokku/dokku) to AWS.